### PR TITLE
Prepare Major release 7.0.0

### PR DESCRIPTION
**Pull request by rohitthakur2590**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -541,7 +559,6 @@ New Modules
 - junos_lag_interfaces - Link Aggregation Juniper JUNOS resource module
 - junos_lldp_global - LLDP resource module
 - junos_lldp_interfaces - LLDP interfaces resource module
-- junos_logging - Manage logging on network devices
```
Comment (NilashishC) [inline CHANGELOG.rst:544]: Should we revert this?

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -46,9 +46,6 @@ releases:
       - description: LLDP interfaces resource module
         name: junos_lldp_interfaces
         namespace: ""
-      - description: Manage logging on network devices
```
Comment (NilashishC) [inline changelogs/changelog.yaml:49]: Same as above.

**Review by NilashishC — CHANGES_REQUESTED** [review]
Can you also remove meta/runtime.yml entries?

File: CHANGELOG.rst (hunk 1 above)
Comment (rohitthakur2590) [reply CHANGELOG.rst:544]: yes I'll update with the reverts.

**Comment by rohitthakur2590** [comment]
recheck

**Pull request merged**